spotbugs/etc/messages.xml

```diff
@@ -9388,6 +9408,62 @@ Using floating-point variables should not be used as loop counters, as they are
       </p>]]>
     </Details>
   </BugPattern>
+  <BugPattern type="AT_NONATOMIC_64BIT_PRIMITIVE">
+    <ShortDescription>This write of this 64-bit primitive variable may not atomic</ShortDescription>
+    <LongDescription>This write of the 64-bit primitive variable "{2.name}" may not atomic</LongDescription>
+    <Details>
+      <![CDATA[
+        <p>
+          The long and the double are 64-bit primitive types, and depending on the Java Virtual Machine implementation assigning a value to them can be treated as two separate 32-bit writes, and as such it's not atomic.
```

Comment:
Should have a comma after "implementation". Also, the sentence should probably be split to avoid double "and". Perhaps change the first comma to a fulls top.